lib/irb/ruby-lex.rb

```diff
@@ -65,68 +65,43 @@ def configure_io(io)
           else
             false
           end
-        else
+        elsif single_line_command?(code)
```

Comment:
I wonder if `single_line_command` related changes could be extracted from the rest?

Reply:
I revert it